src/logging.c

```diff
@@ -135,12 +148,68 @@ void rcutils_logging_console_output_handler(
   }
 
   if (!location) {
-    fprintf(stream, "[%s] [%s]: %s\n", severity_string, name, message_buffer);
-  } else {
-    fprintf(
-      stream, "[%s] [%s]: %s (%s() at %s:%zu)\n", severity_string, name, message_buffer,
-      location->function_name, location->file_name, location->line_number);
+    // TODO(dhood) before merge
+    // location = empty_location;
+  }
+
+  // Process the format string looking for known tokens
+  char token_start_delimiter = '{';
+  char token_end_delimiter = '}';
+  char output_buffer[1024];
+  memset(output_buffer, '\0', sizeof(output_buffer));
+  const char * str = g_rcutils_logging_output_format_string;
+  size_t size = strlen(g_rcutils_logging_output_format_string);
+
+  for (size_t i = 0; i < size; ++i) {
+    if (str[i] != token_start_delimiter) {
+      strncat(output_buffer, str + i, 1);
+      continue;
+    }
+    // Found a token start delimiter: determine if there's a token or not.
+    char token[20];  // the largest known token
```

Comment:
Still not sure what token we're referring to here. Maybe declaring this with a define next to where the tokens are defined would be more future-proof ? Same for the 1024 above and below, maybe a define would be more appropriate

Reply:
thanks, I've seen you comment I just haven't gotten to it yet. I moved this to in progress and will ping once the comments have been addressed

Reply:
Sounds good, it's just because I saw a commit addressing review comment modifying this line that I responded

Reply:
absolutely; it wasn't clear on my part that they weren't all addressed yet